src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx

```diff
@@ -96,7 +96,7 @@ class ConfirmationPage extends React.Component {
               <strong>Confirmation number</strong>
               <br/>
               <span>V-DCCI-3986</span>
-              {/* <span>{response.confirmationNumber}</span> */}
+              {/* <span>{response.claimId}</span> */}
```

Comment:
@SMLuthi @egarreau or @kreek, can you verify this is how the claim ID will be sent from the API?

Reply:
@cvalarida the response will be in the following example format:

```

{
  "data": {
    "attributes": {
      "claim_id": 600130094,
      "end_product_claim_code": "020SUPP",
      "end_product_claim_name": "eBenefits 526EZ-Supplemental (020)",
      "inflight_document_id": 194300
    },
    "id": "",
    "type": "evss_disability_compensation_form_form_submit_responses"
  }
}
```